frontend/public/components/graphs/prometheus-graph.tsx

```diff
@@ -49,7 +50,11 @@ export const PrometheusGraphLink_: React.FC<PrometheusGraphLinkProps> = ({
       : `/dev-monitoring/ns/${namespace}/metrics?${params.toString()}`;
 
   return (
-    <Link to={url} style={{ color: 'inherit', textDecoration: 'none' }}>
+    <Link
+      to={url}
+      aria-label={`View ${title ? title : ''} Graph in Query Browser`}
```

Comment:
I'd change this to simply "View Graph in Query Browser" and not add a `title` prop to this component. It's already being read as the title above the graph. No need for a screen reader to read it twice.

Reply:
Hmm..but that goes against @jessiehuff 's article's recommendations:
"Link text should be unique within a page, should be meaningful **_when read out of context_**, and should help users to know something about their destination if they click on it. Link text such as “Click here” and “More” fail to meet these criteria. Consider the various ways users interact with links"